gateway/handler_error.go

```diff
@@ -20,6 +23,54 @@ const (
 	defaultContentType    = headers.ApplicationJSON
 )
 
+const (
+	OAuthAuthorizationFieldMissing config.TykErrorType = "oauth.auth_field_missing"
+	OAuthBearerTokenMalformed      config.TykErrorType = "oauth.bearer_token_malformed"
+	OAuthKeyNotAuthorised          config.TykErrorType = "oauth.key_not_authorised"
+	OAuthClientDeleted             config.TykErrorType = "oauth.client_deleted"
```

Comment:
Suggest to add Err postfix or prefix, to have consistent notation: `OAuthAuthorizationFieldMissingErr` or `ErrOAuthAuthorizationFieldMissing`